output/forms.eslint.txt

```diff
@@ -122,13 +122,13 @@ The react/require-extension rule is deprecated. Please use the import/extensions
   77:3   error  Prop type `object` is forbidden                           react/forbid-prop-types
   77:16  error  'formContext' PropType is defined but prop is never used  react/no-unused-prop-types
 
-/home/travis/build/Talend/ui/packages/forms/src/Form.js
-  192:3  error  Prop type `object` is forbidden  react/forbid-prop-types
-
 /home/travis/build/Talend/ui/packages/forms/src/templates/FieldTemplate.js
   82:3  error  Prop type `object` is forbidden  react/forbid-prop-types
   83:3  error  Prop type `object` is forbidden  react/forbid-prop-types
 
+/home/travis/build/Talend/ui/packages/forms/src/UIForm/fields/CheckBox/SimpleCheckBox.component.js
+  5:3  error  Form controls using a label to identify them must be programmatically associated with the control using htmlFor  jsx-a11y/label-has-for
+
```

Comment:
We can't set an htmlFor on this `<label>` that wrap the entire checkbox. it will introduce some weird behavior or not working behavior.
The only way to not have this warning is... to suppress it in the file :/